### DISCUSSION TITLE
Can we use use with another git server?

### DISCUSSION THREAD

**Discussion by BuildMonkey** [josh-project/josh#1342] [General]
Hello!

I was wondering if we can set JOSH_REMOTE to another git server other than github.com. I have been reviewing the docs and see github.com in all the exemplars. Is it possible to change to another one?

Thanks in advance.

**Comment by VisualFox** [comment]
Hi there,

I'm able to use my own gitea instance without any issue. From the documentation it seem that as long as the git server can communicate via HTTP you should be fine. @see https://josh-project.github.io/josh/reference/proxy.html
